node/src/GlideClient.ts

```diff
@@ -208,6 +207,10 @@ export class GlideClient extends BaseClient {
      *
      * @see {@link https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#custom-command|Valkey Glide Wiki} for details on the restrictions and limitations of the custom command API.
      *
+     * @param args - A list of `function` arguments and it should not represent names of keys.
+     * @param options - (Optional) See {@link DecoderOption}.
+     * @returns The invoked function's return value.
```

Comment:
```suggestion
     * @returns The executed custom command return value.
```

Reply:
consider updating docs for custom command in other clients too (in another PR)

Reply:
Updated here. Will create a separate task for other clients.